Counterexamples/Phillips.lean

```diff
@@ -385,7 +385,7 @@ section
 
 
 theorem norm_indicator_le_one (s : Set α) (x : α) : ‖(indicator s (1 : α → ℝ)) x‖ ≤ 1 := by
-  simp only [indicator, Pi.one_apply]; split_ifs <;> norm_num
+  simp only [Set.indicator, Pi.one_apply]; split_ifs <;> norm_num
```

Comment:
Is this a (weak) indication that the new one should be protected?

Reply:
Maybe, but in general I think it's fine to have both `A.X` and `B.X` unprotected.